packages/remix-gcloud-functions-gen2/index.ts

```diff
@@ -0,0 +1,4 @@
+import "./globals";
+
+// export type { GetLoadContextFunction, RequestHandler } from "./server";
+export { createRequestHandler, createRemixHeaders, createRemixRequest } from "./server";
```

Comment:
There's no need to export `createRemixHeaders` & `createRemixRequest` from the adapter I think?

```suggestion
export { createRequestHandler } from "./server";
```